### ISSUE TITLE
L1 features not yet in L2

### ISSUE THREAD

**Issue by nem0** [nem0/LumixEngine#1256]
- [x] squirrel
- [x] soft shadows
- [x] terrain
- [x] grass
- [x] transparency
- [x] render selection
- [x] 3d text
- [x] decals
- [x] point lights
- [x] point light shadows
- [x] IK
- [x] ~~GUI~~, ~~Physics~~, Navigation, ~~Audio~~ plugins
- [ ] ~~Data packing~~ Moved to separate issue

**Comment by rokups** [comment]
When will squirrel be implemented? My project is blocked by lack of this feature.

**Comment by nem0** [comment]
@rokups rest assured you will be the first to know
